src/main/java/com/owncloud/android/lib/common/operations/RemoteOperationResult.java

```diff
@@ -482,10 +482,16 @@ public void setSingleData(Object object) {
     }
 
     public ArrayList<Object> getData() {
+        if (!mSuccess) {
+            throw new RuntimeException("Accessing result data after operation failed!");
```

Comment:
Not sure if it is worth the effort but throwing a top-level exception is rather uncommon, not sure if there is an existing, better-fitting Runtime child exception or if one should be created (while it should of course be a RuntimeException)

Reply:
As this normally would not happen, I'd like to skip the additional effort.

Reply:
sure thing!